uint32 RMP_MOTION_CMD_ID= 1280
uint32 RMP_CFG_CMD_ID = 1281
uint8 RMP_CMD_SET_OPERATIONAL_MODE = 5
uint8 TRACTOR_REQUEST = 1
uint8 STANDBY_REQUEST = 2

Header header
uint8 cmd_id
float32 arg1
float32 arg2

